markdown/articles/2021/08/22/Non-blue-fair-decks.md

```diff
@@ -0,0 +1,429 @@
+In games were you can't combo safe in turn 1 or 2 the game can evolve into a staredown while both players dig for interaction. Discard can be very helpfull in situations like this. But be careful, if you play a list with less cantrips, they will sculpt their hand better than you most of the time.  
+In post board games they have two tricks up their sleeve, Blood Moon effects and Ragavan. Try to keep that in mind, but it's nothing match up defining.
+
+#### Post Doomsday 
+
+After you resolved Doomsday only Sneak Attack can kill you. Show and Tell is now very risky for them to cast, because it's just an Oracle cost reducer now, and if you should have no Thassa's Oracle in hand it atleast gives you another turn to win the game. A small trick they can use, they can put Sneak Attack into play with Show and Tell.
+
+## UG Omni-Tell
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Force of Will}} {{!Veil of Summer}} {{!Force of Will}} {{!Cunning Wish}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Surgical Extraction}} {{!Endurance}}{{!Trickbind}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Omni-Tell is a lot slower than it's sister Sneak and Show, since it plays only Show and Tell and no Sneak Attack. It also plays only 4 counterspells most of the time. That's overall very positive for us. Not much protection is needed and we have time to set up, but a lost counter war over Doomsday, can have harsh consequences if they fetch Surgical Extraction from their Sideboard.
+
+#### Post Doomsday 
+
+You can build pretty safe piles that either use their Show and Tell to put Thassa's Oracle into play or just cast it of Cavern of Souls.  
+Some Omni-Tell lists play Trickbind in their Sideboard. Try to use discard spells to get rid of it if necessary.
+
+## Hogaak
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Hedron Crab}} {{!Vengevine}} {{!Cabal Therapy}} {{!Altar of Dementia}}</row>
+
+#### Sideboard
+
+### Gameplay
+
+#### Pre Doomsday
+
+The Hogaak matchup can be pretty tricky, because they have two ways to kill you, either with damage or with mill. They don't even have to completely mill your 60 cards, the pure on board threat of milling us is often enough to make a kill with Doomsday impossible.  
+There are two iterations of Hogaak, a jund version and BUG one. The BUG versions is scarier by far, thanks to Hedron Crab.  
+Speed is the only way to beat Hogaak, they are just to explosive to give them time and you don't want to play against Hedron Crab and a uncracked Fetchlands.  
+They don't play any sideboard hate for Doomsday, but countermagic is still very important to keep them in check and protect your combo.
+
+#### Post Doomsday 
+
+Because Hogaak plays no interaction, you can build Brainstorm piles when ever you can. Should you have to pass the turn, make sure you have answers at your disposal.
```

Comment:
Change "no interaction". 
Crab, Altar, Cabal Therapy and Trophy all fit here.

Reply:
True.
Changed to no interaction on the stack